### ISSUE TITLE
INF loss

### ISSUE THREAD

**Issue by metaluga145** [Mukosame/Zooming-Slow-Mo-CVPR-2020#6]
Hi, guys!

I tried to rerpoduce your results and seems that I'm getting gradient explosion.
After about 100-200 iterations loss gets to INF.

I used config provided by you, is it correct?
How can I fix this issue?

**Comment by Mukosame** [comment]
Hi @metaluga145 , thanks for your interest in our work!
I've never encountered such an issue before. It's definitely not good. Would you like to provide more details about your settings?
Thanks!

**Comment by metaluga145** [comment]
PyTorch: 1.4
CUDA: 10.2

Training dataset: Vimeo90K

**Comment by metaluga145** [comment]
Apparently, one of my GPUs has quite strange behaviour, so I will close this issue :)
Anyway, thank you for your quick response!

**Comment by Mukosame** [comment]
You're welcome, @metaluga145 ! 
I'm sorry for the problems with your GPUs. Hope everything goes well. And plz don't hesitate to let me know if you have any further questions about this project.

**Comment by metaluga145** [comment]
@Mukosame I just have one follow up question: is it okay to get message with warning?
"WARNING: Offset mean is 900.234567, larger than 100"

**Comment by Mukosame** [comment]
It's not OK, it usually implies your dcn is not learning the offset correctly.

**Comment by metaluga145** [comment]
@Mukosame Okay, thanks! I will need to look into that

**Comment by metaluga145** [comment]
@Mukosame Hi there! Could you tell what would be the value of training loss after convergence? Thank you!
